src/elements/quick-note/QuickNote.ts

```diff
@@ -16,62 +14,51 @@ const QUICK_NOTE_VALUE_ACCESSOR = {
     multi: true
 };
 
+declare var CKEDITOR: any;
+
 @Component({
     selector: 'novo-quick-note',
     providers: [QUICK_NOTE_VALUE_ACCESSOR],
     template: `
         <div class="quick-note-wrapper">
-            <textarea class="quick-note-textarea"
-                      [(ngModel)]="basicNote"
-                      (ngModelChange)="onChange($event)"
-                      (keyup)="onKeyUp($event)"
-                      (keypress)="onKeyPress($event)"
-                      (scroll)="onScroll($event)"
-                      (focus)="onFocus($event)"
-                      (blur)="onTouched($event)"
-                      #textArea>
-            </textarea>
-            <div class="quick-note-overlay"
-                [innerHTML]="formattedNote"
-                [attr.placeholder]="placeholder"
-                 #quickNoteOverlay>
-            </div>
+            <textarea #host></textarea>
```

Comment:
Can this just use the CKEditor component that we already have? Would clean up some of the class?